### PR TITLE
remove project.json from shipping projects

### PR DESCRIPTION
**Pull request by rohit21agrawal**
we still had two shipping projects that were using project.json for managing dependencies..converting them into package reference.

### CODE REVIEW

File: build/common.legacy.props [hunk 1]
```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <IsNetCoreProject>false</IsNetCoreProject>
   </PropertyGroup>
+  <ItemGroup Condition="'$(Shipping)' == 'true' AND '$(IsXPlat)' != 'true'">
```
Comment (nkolev92) [inline build/common.legacy.props:6]: This would tag all projects though, woudln't it?

Comment (nkolev92) [reply build/common.legacy.props:6]: Wouldn't this get added through legacy.props -> common.project.props -> common.props

Comment (rohit21agrawal) [reply build/common.legacy.props:6]: no..common.props is only imported for sdk projects... common.project.props is imported everywhere

common.legacy.props is imported for legacy projects

common.legacy.props and common.props import common.project.props , not the other way around.

**Review by nkolev92 — COMMENTED** [review]
Are there new projects that'll get PdbGit but that didn't have it before?

**Comment by rohit21agrawal** [comment]
@nkolev92 yes, all legacy package ref based projects  with Shipping=true

**Review by nkolev92 — APPROVED** [review]

File: build/common.legacy.props (hunk 1 above)
Comment (nkolev92) [reply build/common.legacy.props]: Actually this doesn't work. 
Totally slipped my mind. 

These conditions are not respected and all projects will be tagged with pdbgit. 

Don't know if that's a problem. 
Let's discuss offline.

**Pull request merged**